source/guides/network/wifi.rst

```diff
@@ -102,8 +104,119 @@ TUI (Text-based User Interface)
 
    .. figure:: /_figures/wifi/nmtui_5.png
 
+wpa_supplicant
+**************
+
+Make sure NetworkManager is either disabled or not installed before using
+wpa_supplicant.
+
+.. code-block:: bash
+
+   sudo systemctl stop NetworkManager.service
+
+#. Create a ``wpa_supplicant`` configuration directory.
+
+   .. code-block:: bash
+
+      sudo mkdir /etc/wpa_supplicant
+
+#. Create a minimal :file:`/etc/wpa_supplicant/wpa_supplicant.conf` and add the
+   following:
+
+   .. code-block:: bash
+
+      ctrl_interface=/run/wpa_supplicant
+      update_config=1
+
+#. Determine your wireless interface name.
+
+   .. code-block:: bash
+
+      iw dev
+
+#. Complete the configuration process as ``root``.
+
+   .. code-block:: bash
+
+      sudo su
+      wpa_supplicant -B -i $INTERFACE -c /etc/wpa_supplicant/wpa_supplicant.conf
+
+#. Use :command:`wpa_cli` (interactive mode) to scan for available networks.
+
+   .. code-block:: bash
+
+      > scan
+      OK
+      <3>CTRL-EVENT-SCAN-STARTED
+      <3>CTRL-EVENT-SCAN-RESULTS
+      > scan_results
+      bssid / frequency / signal level / flags / ssid
+      00:xx:xx:73:7b:46 5180 -55 [WPA2-PSK-CCMP][ESS] Network1
+      00:xx:xx:5d:d9:23 2412 -47 [RSN--CCMP][MESH] 1137e9
+      00:xx:xx:73:7b:43 2412 -49 [RSN--CCMP][MESH] 1137e9
+      00:xx:xx:5d:d9:25 2412 -47 [WPA2-PSK-CCMP][ESS] Network1
+      00:xx:xx:37:25:05 2412 -49 [WPA2-PSK-CCMP][ESS] Network1
+      00:xx:xx:73:7b:45 2412 -59 [WPA2-PSK-CCMP][ESS] Network1
+      00:xx:xx:83:fa:6a 2437 -57 [WPA-EAP-CCMP+TKIP][WPA2-EAP-CCMP+TKIP][ESS]
+      00:xx:xx:83:fa:70 5240 -76 [WPA2-EAP-CCMP][ESS] Network2
+      00:xx:xx:4f:e9:2c 2412 -67 [WPA2-PSK-CCMP][ESS][P2P] Printer
+      00:xx:xx:af:fe:3e 5765 -79 [WPA2-PSK-CCMP][ESS] Network3
+      00:xx:xx:e9:eb:29 2412 -76 [WPA2-PSK-CCMP][ESS] Network4
+      00:xx:xx:26:4a:b9 2412 -79 [WPA2-PSK-CCMP][ESS][P2P] Printer2
+      00:xx:xx:b9:0d:d4 2462 -79 [WPA2-PSK-CCMP][ESS] Network5
+
+#. Set up your network connection.
```

Comment:
Add something like "In this example the network name is Network1 and the password is "Network1Password". 

We also need to call out that this saves the password in plaintext in the `/etc` configuration file. We should point them to `wpa_passphrase`. I'd like to avoid adding another "fork" in the reading flow. I'd like to  deferring to the manpage, but we don't package the man pages for wpa_supplicant.conf. I can look into getting that done, but for now let's at least mention the security consideration.

Reply:
Also, we should recommend `chmod 600` on this file.